src/create-route-middleware/index.js

```diff
@@ -3,15 +3,20 @@ import getIsEnabled from '../utils/get-is-enabled';
 import updateFeatures from '../utils/updateFeaturesWithParams';
 import { parse } from 'url';
 
-const handleResponse = (res, isEnabled) =>
+const setStatus = (res, isEnabled) =>
   isEnabled ? res.status(200) : res.status(404);
 
-const createRoute = (features, featureName) => (req, res, next) => {
+const createRoute = (features, { requiredFeature, ...methods }) => (req, res, next) => {
   const parsedUrl = parse(req.url, true);
   const { search } = parsedUrl;
   const updatedFeatures = updateFeatures(features, search);
+  setStatus(res, getIsEnabled(updatedFeatures, requiredFeature));
+
+  const handler = methods[req.method.toLowerCase()];
+  if (handler !== void 1 && typeof handler === 'function') {
```

Comment:
Say what you mean: `undefined` instead of `void 1`. If somebody has clobbered `undefined` on purpose, trust me -- the app is never going to get to this code anyway. ;)

Reply:
Updated

Reply:
Also, in ES6...

<img width="319" alt="screen shot 2017-11-15 at 10 24 39 am" src="https://user-images.githubusercontent.com/364727/32853179-5064a364-c9ef-11e7-8a4f-961e6cf143eb.png">

Reply:
^^ strict mode magic

Reply:
nice!